Lib/test/test_time.py

```diff
@@ -163,6 +163,12 @@ def test_sleep(self):
         self.assertRaises(ValueError, time.sleep, -1)
         time.sleep(1.2)
 
+    def test_sleep_invalid(self):
+        with self.assertRaisesRegex(
+                TypeError, 'an integer or a float is required \(got type str\)') as te:
+            time.sleep('foo')
+
+
```

Comment:
It looks like all of the other tests have just one blank line between them. Please delete one of the lines.